ALevelHomework5/Scripts/Class1.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+public enum LogType
```

Comment:
It's better to move this enum to a separate file, but good that you decided to use it